### DISCUSSION TITLE
Add passport-local or even passport-local-passwd

### DISCUSSION THREAD

**Discussion by schemen** [outline/outline#1944] [Ideas]
Hi!

WIth https://github.com/outline/outline/pull/1934 being implemented soon, how about adding local authentication?

Or add a guide on how to add it :) I'm afraid I'm not fluent in javascript, otherwise I would gladly contribute!

Nevertheless, I'll continue to have a look in case I somehow manage

**Comment by tommoor** [comment]
This is covered by several issues, latest is here: 
https://github.com/outline/outline/issues/1881

**Reply by schemen** [reply]
Ah cool, sorry my search-fo seems to have left me :>

Thanks! I look forward to it!
